src/status_im/contexts/communities/events.cljs

```diff
@@ -399,22 +399,24 @@
      [:json-rpc/call :schema.common/rpc-call]]]])
 
 (rf/reg-event-fx :communities/get-revealed-accounts-success
- (fn [{:keys [db]} [community-id on-success revealed-accounts-js]]
+ (fn [{:keys [db]} [community-id on-success latest-request-to-join-js]]
    (when-let [community (get-in db [:communities community-id])]
-     (let [revealed-accounts
-           (reduce
-            (fn [acc {:keys [address] :as revealed-account}]
-              (assoc acc address revealed-account))
-            {}
-            (data-store.communities/<-revealed-accounts-rpc revealed-accounts-js))
-
+     (let [request-to-join (js->clj latest-request-to-join-js :keywordize-keys true)
+           revealed-accounts (
+                              reduce
+                              (fn [acc {:keys [address] :as revealed-account}]
+                                (assoc acc address revealed-account))
+                              {}
+                              (js->clj (:revealedAccounts request-to-join) :keywordize-keys true))
```

Comment:
It shouldn't be needed to convert again from JS to CLJ because `js->clj` is recursive and if you remove `:json-response true` from event `:communities/get-revealed-accounts`, then you can eliminate this transformation too.

Reply:
Fixed.